src/Setup.php

```diff
@@ -652,7 +661,18 @@ protected function loadDefaultACFFields(){
       				),
       			),
       		),
-      	),
+		);
+
+      acf_add_local_field_group(array (
+      	'key' => 'group_59145820f23d4',
+      	'title' => __('Event', 'clarkson-event-manager'),
+      	'fields' => array_merge(
+			apply_filters('clarkson_event_manager_tab_date', $tab_date) ? $tab_date : array(),
+			apply_filters('clarkson_event_manager_tab_location', $tab_location) ? $tab_location : array(),
+			apply_filters('clarkson_event_manager_tab_information', $tab_information) ? $tab_information : array(),
+			apply_filters('clarkson_event_manager_tab_ticket_registration', $tab_ticket_registration) ? $tab_ticket_registration : array(),
+			apply_filters('clarkson_event_manager_tab_contact', $tab_contact) ? $tab_contact : array()
+		),
```

Comment:
Als je tabs uitzet, komen er lege arrays in de fields te staan.
Ik denk dat (eventuele toekomstige) versies van acf dat niet nice vinden.

Doe je array merge hierboven al en stop het resultaat in een variabele.
Doe daarna een array_filter eroverheen, om de lege arrays eruit te vissen.
En dat resultaat geef je dan mee aan $fields.

Reply:
Verder denk ik dat het geen kwaad kan nog even kort naar tabbing en spacing te kijken